package.json

```diff
@@ -68,7 +72,7 @@
     "name": "console-plugin-template",
     "version": "0.0.1",
     "displayName": "OpenShift Console Plugin Template",
-    "description": "Template project for OpenShift console plugins. Edit package.json to change this message and the plugin name.",
+    "description": "Template project for OpenShift Console dynamic plugins. Edit package.json to change this message and the plugin name.",
```

Comment:
Let's not make the distinction between static and dynamic plugins since static plugins are internal and not something someone outside of the project needs to know. We should just call them plugins.

Reply:
I'd argue that calling them dynamic plugins is more expressive and descriptive.

That said, I'll make the suggested change.